From ae6f7bef78ec2ba71e6009257b736226049c437b Mon Sep 17 00:00:00 2001
From: Dhina17 <dhinalogu@gmail.com>
Date: Sat, 16 Nov 2024 16:06:33 +0530
Subject: [PATCH 22/35] SystemUI: Remove udfps dim on unconfigureDisplay()

on unconfigureDisplay(), it removes the udfps
hbm illumination dot so that dim also should be removed.

Change-Id: Ic2a8801860eec260da0f6b22c26a0fa0065e9e84
---
 .../src/com/android/systemui/biometrics/UdfpsController.java     | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/SystemUI/src/com/android/systemui/biometrics/UdfpsController.java b/packages/SystemUI/src/com/android/systemui/biometrics/UdfpsController.java
index 7a82ebe2db80..4684faf3a9d2 100644
--- a/packages/SystemUI/src/com/android/systemui/biometrics/UdfpsController.java
+++ b/packages/SystemUI/src/com/android/systemui/biometrics/UdfpsController.java
@@ -876,6 +876,7 @@ public class UdfpsController implements DozeReceiver, Dumpable {
         if (udfpsView.isDisplayConfigured()) {
             udfpsView.unconfigureDisplay();
         }
+        mOverlay.setDimAmount(0f);
     }
 
     /**
-- 
2.49.0

